data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,227 @@ mission "Remnant: Visit to Paradise 1"
+			
+
+
+mission "Remnant: Shattered Light 3"
+	invisible
+	name "Ghost Touch"
+	description "Help the Remnant experiment on the Shattered Light."
+	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 cargo."
+	cargo "equipment" 19
+	passengers 13
+	source "Viminal"
+	destination "Far Monad"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Lucina
+		ship "Dromedary Specter" "???"
+		dialog `It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
+	on offer
+		conversation
+			`A group of people huddled around a hologram catches your attention, drawing your eyes to Rauch who gestures for you to join them. She introduces you to each person, before thanking you: "I would just like to start by offering our thanks for your assistance, Captain <first>. You have sharp eyes to pick this up as early as you did." She gestures to the hologram of the ship floating in front of you. "Other scouts reported it shortly thereafter, but you spotted it first."`
+			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the oldest mysteries here is what has happened to it, and what we should do about it."`
+			`	Another Remnant steps forward and the hologram changes to show a scene from a cave with a large object. Other Remnant appear to aim some equipment at it, and after a moment or two, the object expands to several times its original size. "These are recordings from our explorations of the ruins, demonstrating the decompression of a Maudrakoi artifact. While not identical, these artifacts generate similar readings to those recorded during your scan of the Shattered Light.`
+			`	"As such, our proposition is to use the same equipment we used for decompression to stabilize the Shattered Light's weft field, and, if possible, bring it fully into synchronicity once again." She nods firmly, clapping her hands together to make the hologram vanish.`
+			`	Ruach takes the lead again. "So, Captain, we will need you to bring us in close to the Shattered Light so we can give this a try. Our sentries have reported seeing it in the systems near Coluber, so it may take some searching."`
+			choice
+				`	"Looks good. Start loading the <ship>!"`
+					accept
+				`	"Wait, 'Maudrakoi'?"`
+			`	The Remnant who showed the video looks startled, and then embarrassed. "My apologies, I did not realize you had not been familiarized with them." They look around the room, then gesture and their hologram shifts to show a large room with many shelves around the edge and several large objects within it, including some that look like point defense turrets.`
```

Comment:
Any particular reason gender neutral prnouns are used here when feminine ones are used both before and affter this line? Also, this hologram was just turned off.
```suggestion
			`	The Remnant who showed the video looks startled, and then embarrassed. "My apologies, I did not realize you had not been familiarized with them." She look around the room, then gesture and the hologram returns, now showing a large room with many shelves around the edge and several large objects within it, including some that look like point defense turrets.`
```

Reply:
Not a strong reason, no. I tend to drift towards the neutral when the speaker's gender isn't an important detail, or when I prefer to shift the focus on the description.